src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -65,7 +65,7 @@ class PaymentAmount(BaseStaticVariable):
     def get_initial_value(self, submission: Submission | None = None):
         if submission is None:
             return None
-        return str(submission.payments.sum_amount())
+        return float(submission.payments.sum_amount())
```

Comment:
This seems to be the [documented way](https://docs.python.org/3/library/decimal.html#quick-start-tutorial)
![image](https://github.com/open-formulieren/open-forms/assets/65306057/ff399a8e-7f43-4170-b2da-aaa6637fcf3f)
: